### DISCUSSION TITLE
Forking is not allowed (when forking a forked repository)

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#146859] [Mobile] [answered]
GitHub Mobile beta for Android

Open https://github.com/mahabaleshwars/java-sample in GitHub Mobile and select the fork icon

![Screenshot_20241211-075755.png](https://github.com/user-attachments/assets/1520658b-75f4-48cd-bba8-587425879f27)

Compare in browser
![Screenshot_20241211-075820.png](https://github.com/user-attachments/assets/eb6bf1ae-9b27-4880-94c7-7d6ed76e1769)

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

Thanks for your report ❤️ We can reproduce the issue on our end and will be releasing a fix for forking a forked repository asap 🙇 

Cheers!
